components/brave_welcome_ui/components/screens/searchBox.tsx

```diff
@@ -35,14 +57,28 @@ export default class SearchEngineBox extends React.PureComponent<Props, {}> {
       >
         <WelcomeSearchImage />
         <Title>{getLocale('setDefaultSearchEngine')}</Title>
-        <Paragraph>{getLocale('chooseSearchEngine')}</Paragraph>
-          <Button
-            level='primary'
-            type='accent'
-            size='large'
-            text={getLocale('search')}
-            onClick={onClick}
-          />
+        <Paragraph>{bodyText}</Paragraph>
+          <SelectGrid>
+            <SelectBox onChange={this.onChangeDefaultSearchEngine}>
+              <option key={0} value=''>{getLocale('selectSearchEngine')}</option>
+              {searchProviders.map((provider, index) =>
+                <option
+                  key={index + 1}
+                  value={provider.value}
+                >
+                  {getLocale(provider.name)}
```

Comment:
@cezaraugusto since this list is populated based on API returns should I just leave this as `provider.name` rather than `getLocale(provider.name)`

Reply:
we can remove `getLocale` here since we don't translate their names